fixing edit creation options in raster miscellaneous to align with #3755 (#3836)
qgis/QGIS-Documentation: source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst

``Additional creation options``
  Optional

>   Allows to add one or more optional creation options that can be used to control

Reviewer: ```suggestion
  Allows to add one or more creation options that can be used to control
```

``Additional creation options``
  Optional

>   Allows to select or add a profile to be used on the raster creation, see

Reviewer: "Allows to select or add a profile to be used for the raster creation" ("on" -> "for")

Reviewer 2: I think it's the other way. what matters is the parameters you can add, not the profiles which are some convenient GUI tools, not useful for command line users for example. When you select a profile, it populates the call with those parameters (see command line frame at the bottom) but you can also directly use the `+` button to add parameters.
I'd then keep the previous description tweaked with the [official doc of this parameter](https://gdal.org/programs/gdal_merge.html#cmdoption-gdaladdo-co).
```suggestion
  Allows to add one or more optional creation options that can be used to control
  particulars (colorimetry, block size, file compression...) about the file
  created. For convenience, you can rely on predefined profiles (see
```

Author: ... about the file to be created?

>   :ref:`GDAL driver options section <gdal_createoptions>`

Reviewer 2: ```suggestion
  :ref:`GDAL driver options section <gdal_createoptions>`).
```

>   Predefined profiles are listed.

Reviewer 2: ```suggestion
```

Author: I disagree, but I changed as you suggested

Reviewer 2: Thanks... sorry but you disagree with the removal of this line (I think it's somehow implied in the suggested rewording) or with removing the whole profiles description?

Author: With the removing of the whole profiles description. But, it's done.

>   ``Profile`` [enumeration]

Reviewer 2: And I'd remove mention of profiles from here...to the end. If someone wants to know what the profiles are, the gdal create options might explain that and help to customize. But these profiles are not values for the algorithm (and vary or vanish if my default format is not geotiff) and will be cumbersome when we update the description structure with the corresponding python calls.
